### PR TITLE
Added via keymap for bpiphany/pegasushoof

### PR DESCRIPTION
**Pull request by C4illin**
## Description

Added VIA keymap and changed info.json and config.h for 2015 to include device manufacturer, name and USB vid and pid. The changes to 2015 keyboard should cause no conflicts to existing users.

The VIA keymap matches the default keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bpiphany/pegasushoof/2015/config.h [hunk 1]
```diff
@@ -17,6 +17,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4245  // "BE" (Bathroom Epiphanies)
+#define PRODUCT_ID      0x6050  // "`p"
+#define DEVICE_VER      1.0
+#define MANUFACTURER    "Filco"
+#define PRODUCT         "Majestouch TKL The Pegasus Hoof 2015"
+
```
Comment (zvecr) [inline keyboards/bpiphany/pegasushoof/2015/config.h:26]: This should not be added back, any required changes must instead be made to `info.json`
```suggestion
```

File: keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+CUSTOM_MATRIX = yes         # Custom matrix file for the Pegasus Hoof due to the 2x74HC42
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no        # RGB underlight.
+
```
Comment (zvecr) [inline keyboards/bpiphany/pegasushoof/keymaps/via/rules.mk:15]: This should only contain overrides, not a copy/paste from the keyboard level
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define KM_QWERTY  0
+#define KM_MEDIA   1
```
Comment (drashna) [inline keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c:19]:
```suggestion
enum custom_layers {
    KM_QWERTY,
    M_MEDIA,
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum custom_layers {
+    KM_QWERTY,
+    M_MEDIA,
```
Comment (drashna) [inline keyboards/bpiphany/pegasushoof/keymaps/via/keymap.c:20]: Seems like the K got truncated? 
```suggestion
    KM_MEDIA,
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/pegasushoof/2015/info.json [hunk 5]
```diff
@@ -1,5 +1,11 @@
 {
-    "keyboard_name": "Majestouch TKL \\\\w The Pegasus Hoof 2015",
+    "keyboard_name": "Majestouch TKL The Pegasus Hoof 2015",
+    "manufacturer": "Filco",
+    "usb": {
+        "vid": "0x4245",
+        "pid": "0x6050",
+        "device_version": "1.0.0"
+    },
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/2015/info.json:8]: If this isn't any different from what's in `keyboards/bpiphany/pegasushoof/info.json`, no need for it to be in this file.
```suggestion
```

File: keyboards/bpiphany/pegasushoof/keymaps/via/config.h [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```
Comment (waffle87) [inline keyboards/bpiphany/pegasushoof/keymaps/via/config.h:18]:
```suggestion
#define DYNAMIC_KEYMAP_LAYER_COUNT 2
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/pegasushoof/keymaps/via/config.h (hunk 6 above)
Comment (C4illin) [reply keyboards/bpiphany/pegasushoof/keymaps/via/config.h:18]: But it works with 3?

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by C4illin** [comment]
Is there anything I need to do?

**Pull request merged**